Sources/SwiftParserDiagnostics/SyntaxExtensions.swift

```diff
@@ -71,7 +71,7 @@ extension SyntaxProtocol {
   /// diagnostic message), return that.
   /// Otherwise, return a generic message that describes the tokens in this node.
   var shortSingleLineContentDescription: String {
-    let contentWithoutTrivia = self.withoutLeadingTrivia().withoutTrailingTrivia().description
+    let contentWithoutTrivia = self.with(\.leadingTrivia, []).with(\.trailingTrivia, []).description
```

Comment:
Yeah, it seems wasteful but it seems like a fairly narrow use-case and I don’t want to blow up our API surface for such use cases.

Reply:
Doesn't seem *that* narrow. Can leave for now, we can always add later.

Reply:
Drive-by comment: it's useful for SwiftLint, we currently have 13 occurrences of `withoutTrivia().description` https://github.com/search?q=repo%3Arealm%2FSwiftLint%20withoutTrivia().description&type=code

Maybe there's a better way to accomplish this, but that's what we're doing now.

Reply:
OK, you convinced me. I added `trimmedDescription`. I decided against naming it `descriptionWithoutTrivia` because that sounds like we would also remove trivia in the middle of the node.